### PR TITLE
Added ListAllFulfillmentOrders, ListAllFulfillmentOrdersByNextToken, …

### PR DESCRIPTION
**Pull request by KleinerNull**
…GetFulfillmentOrder to the OutboundShipments class

I needed to write this for my own so why not share this for the rest of us ;) Now atleast the OutboundFulfillment API is usable.

### CODE REVIEW

File: mws/mws.py [hunk 1]
```diff
@@ -614,6 +614,21 @@ def list_inventory_supply_by_next_token(self, token):
...
+        return self.make_request(data, "POST")
+
+    def list_all_fulfillment_orders_by_next_token(self, next_token):
+        data = dict(Action='ListAllFulfillmentOrdersByNextToken',
+                    NextToken=next_token)
+        return self.make_request(data, "POST")
+
+    def get_fulfillment_order(self, seller_fulfillment_order_id):
+        data = dict(Action='GetFulfillmentOrder',
+                    SellerFulfillmentOrderId='seller_fulfillment_order_id')
```
Comment (jameshiew) [inline mws/mws.py]: Should be `seller_fulfillment_order_id` instead of `'seller_fulfillment_order_id'` :+1:

**Comment by jameshiew** [comment]
Hey @KleinerNull, is it good for me to integrate your PR into the repo I've set up here (https://github.com/jameshiew/mws)? Then these changes can be pushed to the `mws` pip package

File: mws/mws.py (hunk 1 above)
Comment (KleinerNull) [inline mws/mws.py]: Yeah, you're right :D

**Comment by KleinerNull** [comment]
@jameshiew Yes you can integrate it into the other repo. More funtionality in the pip package sounds nice.

**Comment by jameshiew** [comment]
Great!

**Comment by czpython** [comment]
Thanks for the pr.
Closing as this project is no longer maintained.
Please head over to https://github.com/python-amazon-mws/python-amazon-mws.

**Pull request closed without merging**